tmt/steps/provision/localhost.py

```diff
@@ -22,8 +22,12 @@ def _prepare_ansible(self, what):
         """ Run ansible on localhost """
         # Playbook paths should be relative to the metadata tree root
         playbook = os.path.join(self.step.plan.run.tree.root, what)
+        # Prepare verbose level based on the --debug option count
+        level = len(self.opt('debug'))
```

Comment:
seems this is repeated in all plugins, could this be part of utils or something?

Reply:
First I was thinking about storing number of given options but then I thought that `len(self.opt('debug'))` is not that complicated. But we can adjust the content so that it holds integer instead of list if `(True, True, True)`. Then we could do here:

```python
verbose = ' -' + self.opt('debug') * 'v' if self.opt('debug') else ''
```

Do you think this would be better?

Reply:
Updated to always return debug/verbose level as an `int`.